ansible/tasks/setup-postgres.yml

```diff
@@ -139,6 +139,21 @@
     group: postgres
   when: debpkg_mode or nixpkg_mode
 
+- name: Check if psql_version is psql_15
+  set_fact:
+    is_psql_15: "{{ psql_version in ['psql_15'] }}"
+
+- name: create placeholder pam config
+  file:
+    path: '/etc/pam.d/{{ item }}'
+    state: touch
+    owner: postgres
+    group: postgres
+    mode: 0664
+  with_items:
+    - 'postgresql'
```

Comment:
why loop over a single item? loops are inherently slow

Reply:
half formed idea, thought there might be more files. Good catch, I'll update